l1-contracts/test/fee_portal/depositToAztecPublic.t.sol

```diff
@@ -81,6 +81,8 @@ contract DepositToAztecPublic is Test {
     uint256 amount = 100 ether;
     uint256 expectedIndex = 2 ** Constants.L1_TO_L2_MSG_SUBTREE_HEIGHT;
 
+    // The purpose of including the function selector is to make the message unique to that specific call. Note that
+    // it has nothing to do with calling the function.
```

Comment:
I remember finding the selector on line 89 confusing so I decided to include the comment here.